### PR TITLE
Tophat Hinge for V0

### PR DESCRIPTION
**Pull request by MSiemons**
Tophat Hinge for V0

  * [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This mods add a hinge for the V0 tophat. Almost all parts of the original tophat can be reused. 

#### How was it tested? 
Printed and tested on my V0.1

#### Any background context you want to provide?
I really like the aesthetic of the tophat, but wanted easy acces to the gantry/extruder.

#### Screenshots (if appropriate)
TopHathinge_render2

Tophat_Hinge

Tophat_Closed

Tophat_Open

#### Further notes
First time commiting to VoronDesign! Please let me know if I missed a step/rules :).
Designed for 3mm panels.

### CODE REVIEW

**Comment by MSiemons** [comment]
I believe I implemented the pull request correctly.
Looking forward to any feedback!

**Comment by engpedroamorim** [comment]
does it fit 3mm panels?

**Comment by MSiemons** [comment]
> does it fit 3mm panels?

Yes! I added that in the description. Thanks!

File: printer_mods/MSiemons/TophatHingeV0.1/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Printed and tested: the tophat is relatively stable and stays in the open position.
+
+Requirements:
+
+4 x M3x20
+
+6 x M3x12
+
+![image](https://user-images.githubusercontent.com/38942136/126869895-856b895a-9f4e-4b0c-8633-0dcc69066b98.png)
```
Comment (KiloQubit) [inline printer_mods/MSiemons/TophatHingeV0.1/readme.md:15]: Please upload all the images into your mod directory and reference them from there.  I.e. if you upload this one as `printer_mods/MSiemons/TophatHingeV0.1/image.png`, you can reference it as

    image

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

`Hinge_panel_{left,right}.stl` have unsupported overhangs:
Screen Shot 2021-08-05 at 18 46 24

Is `Lower_Corner_adapter_2_and_4_Clip_x2.stl` the same as `Lower_Corner_adapted_2_and_4_Clip_x2.stl` but in an unprintable orientation?

Please also see README changes below.

**Comment by Shteeeeeve** [comment]
@MSiemons -- I saw another mod out there for a hinge that you could drill into the existing tophat and put in a headset -- I like your design -- but would be cool to have an option to retro an existing tophat, especially for those who printed a unibody tophat.

File: printer_mods/MSiemons/TophatHingeV0.1/readme.md (hunk 1 above)
Comment (MSiemons) [reply printer_mods/MSiemons/TophatHingeV0.1/readme.md]: Thanks for the suggestion! All images are now also in a in a folder.

Comment (MSiemons) [reply printer_mods/MSiemons/TophatHingeV0.1/readme.md]: "Is Lower_Corner_adapter_2_and_4_Clip_x2.stl the same as Lower_Corner_adapted_2_and_4_Clip_x2.stl but in an unprintable orientation?"

Yes - forgot to remove it. Sorry.

Regarding the overhang: not sure how to interpret your screenshot, but Ive modified the part slightly to correct for the (very small) overhang :)

Comment (MSiemons) [reply printer_mods/MSiemons/TophatHingeV0.1/readme.md]: Update on the overhang: I now realize my mistake. Ive modified the part correctly.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**